packages/tenant-process/src/model/domain/errors.ts

```diff
@@ -24,8 +24,11 @@ export const errorCodes = {
   certifiedAttributeAlreadyAssigned: "0015",
   attributeVerificationNotAllowed: "0016",
   verifiedAttributeSelfVerificationNotAllowed: "0017",
-  mailNotFound: "0018",
-  mailAlreadyExists: "0019",
+  attributeRevocationNotAllowed: "0018",
+  attributeAlreadyRevoked: "0019",
+  verifiedAttributeSelfRevocationNotAllowed: "0020",
+  mailNotFound: "0021",
+  mailAlreadyExists: "0022",
```

Comment:
Please bring these two to the previous position (and numbering), so they are not shown as changed